library/src/meta/outline.rs

```diff
@@ -219,13 +223,21 @@ impl Show for OutlineElem {
                 seq.push(HElem::new(Fr::one().into()).pack());
             }
 
+            let page_numbering = vt
+                .introspector
+                .page_numbering(location)
+                .cast::<Option<Numbering>>()
+                .expect("should be a numbering")
+                .unwrap_or_else(|| {
+                    Numbering::Pattern(NumberingPattern::from_str("1").unwrap())
+                });
```

Comment:
can'be an option anymore, so you can just write `.cast::<Numbering>().unwrap()`. personally, I'm not a big fan of expect, it rarely helps more since unwrap gives you the line number nowadays.